_posts/2019-08-18-view-encapsulation-w-angularze.md

```diff
@@ -0,0 +1,426 @@
+---
+layout:    post
+title:     "View Encapsulation w Angularze - czyli o kapsułkowaniu słów kilka"
+published: true
+date:     2019-08-18 08:00:00 +0100
+author:    mhoja
+tags:
+    - angular
+    - view encapsulation
+    - kapsułkowanie
+---
+
+Tworząc komponenty w Angularze mamy możliwość zarządzania kapsułkowaniem styli - czyli tym jak style z jednego komponentu wpływają na inne komponenty.  
+Zanim omówimy kapsułkowanie, wtyjaśnijmy w kilku słowach czym jest **Shadow DOM**.
```

Comment:
>wtyjaśnijmy

wyjaśnijmy

Reply:
Jak czytałem artykuł to strasznie mnie irytowało to wytłuszczone "Shadow DOM". Dodatkowo uspójnij pisownię - raz masz "ShadowDom" a raz "Shadow DOM"

Reply:
Spokojnie, jeszcze nie było przepuszczone przez spell checker i miałem zamiar przeczytać to w całości po skończeniu :)

Reply:
done